components/public-api/gitpod/v1/workspaces.proto

```diff
@@ -0,0 +1,277 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/public-api/v1";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/field_mask.proto";
+import "gitpod/v1/pagination.proto";
+
+service WorkspacesService {
```

Comment:
How can one get an owner token?

Reply:
I see it is from `GetActiveWorkspaceInstance` should work.

Reply:
Right now it would just be part of the workspace instance. I.e. we don't treat that token any different from the other data.

Considering our authorisation model (function scopes), we might want to reconsider that though.

Reply:
I've introduced a `GetWorkspaceInstanceOwnerToken` call and removed the owner token from the workspace instance status.